mqclient/broker_clients/rabbitmq.py

```diff
@@ -23,8 +23,26 @@
     Sub,
 )
 
+StrDict = Dict[str, Any]
+
 LOGGER = logging.getLogger("mqclient.rabbitmq")
 
+# url parsing constants
+HUMAN_PATTERN = "[abc://][USER[:PASS]@]HOST[:PORT][/VIRTUAL_HOST]"
```

Comment:
Maybe instead of `abc://` it should be `SCHEME://` as that's the technical term.